### DISCUSSION TITLE
Access to my OWN project denied

### DISCUSSION THREAD

**Discussion by Wincify** [AntlerVC/firetable#1271] [Support / Q&A] [answered]
<img width="638" alt="Screenshot 2023-05-25 at 11 16 27" src="https://github.com/rowyio/rowy/assets/117103810/03a38e47-ae3c-48e3-94fd-c5418a734187">

**Comment by shamsmosowi** [comment, marked as answer]
did you create the project recently?

**Comment by mkraehe** [comment]
The same happened to me. Do I need to wait for a bit or how can I get access to my data?

**Comment by bofeiw** [comment]
Closing this issue as outdated. 

For access issues or account related issues in the managed Rowy app, please email us at support@rowy.io and provide relevant information or the steps you took that could help us reproduce the issue.
